EXILED/Exiled.CustomItems/Events/MapHandler.cs

```diff
@@ -18,7 +18,7 @@ internal sealed class MapHandler
         /// <inheritdoc cref="Exiled.Events.Handlers.Map.Generated"/>
         public void OnMapGenerated()
         {
-            Timing.CallDelayed(0.5f, () => // Delay its necessary for the spawnpoints of lockers and rooms to be generated.
+            Timing.CallDelayed(0.8f, () => // Delay its necessary for the spawnpoints of lockers and rooms to be generated.
```

Comment:
You can remove timing entirely and use `Server.WaitingForPlayers` instead.

Reply:
Thanks n-n